package.json

```diff
@@ -1,8 +1,35 @@
 {
   "name": "ethereumjs-vm",
   "version": "2.3.3",
-  "description": "an ethereum VM implementation",
-  "main": "index.js",
+  "description": "An Ethereum VM implementation",
+  "main": "dist/index.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "coverage": "istanbul cover ./tests/tester.js -- -s",
```

Comment:
This is lacking `build:dist`.

Reply:
No, it's not, this is explained above. I'll cite here:

"Update: After some experimentation it turned out that Coveralls showing the files covered couldn't be combined with the desired property of not having the dist folder in Git by excluding the folder in .gitignore. So coverage is now run against the lib folder. Which also makes sense."

Reply:
Ah I see, if `--dist` is specified it runs on the `dist` directory, otherwise on `lib`.